dump eth headers and libs-rs/eth-client test use dumped data

ailisp opened the pull request:
Address #58

nearmax commented on environment/commands/dump-eth-headers/index.js:
```diff
@@ -0,0 +1,31 @@
+    console.log('there')
+    console.log(path)
+    console.log(endBlock)
+    for (let b = startBlock; b <= endBlock; b++) {
```
Please add functionality of when `endBlock` is not specified to use the most recent block.

ailisp replied:
This was not the upto date one, i'm doing some cleanup and undo changes in ethrelay (which was using the subscribe method)

nearmax commented on environment/commands/dump-eth-headers/index.js:
```diff
@@ -0,0 +1,31 @@
+
+class DumpETHHeaders {
+  static async execute({path, startBlock, endBlock, ethNodeUrl}) {
+    console.log('here')
```
Please remove debugging prints.

ailisp commented:
To test:
```
node index.js dump-eth-headers --eth-node-url wss://mainnet.infura.io/ws/v3/$INFURA_PROJECT_ID --path /tmp/data --start-block=-10

# in lib-rs
ETH_HEADER_DIR=/tmp/data cargo test --package eth-client --lib -- tests::predumped_block_can_be_added --exact --nocapture
```

nearmax commented on environment/commands/dump-eth-headers/index.js:
```diff
@@ -0,0 +1,45 @@
+    } else {
+      startBlock = Number(startBlock);
+    }
+    console.log(`Download block ${endBlock} down to ${startBlock} to ${path}, totally ${endBlock - startBlock + 1} blocks`)
```
```suggestion
    console.log(`Downloading block ${endBlock} down to ${startBlock} to ${path}. ${endBlock - startBlock + 1} blocks in total.`)
```

ailisp commented:
@nearmax addressed your comment. Also I found if blocks are more than 20-30, get error of:
```
thread 'tests::predumped_block_can_be_added' panicked at 'called `Result::unwrap()` on an `Err` value: HostError(GasLimitExceeded)', /Users/bo/.cargo/registry/src/github.com-1ecc6299db9ec823/near-bindgen-0.4.2/src/environment/mocked_blockchain.rs:133:9
```
How to increase prepaid gas in this test?
